codegen/smithy-aws-swift-codegen/src/test/kotlin/software/amazon/smithy/aws/swift/codegen/awsrestjson/AWSRestJson1ProtocolGeneratorTests.kt

```diff
@@ -58,7 +58,7 @@ extension ExplicitBlobInput {
         val contents = getClientFileContents("Sources/Example", "ExampleClient.swift", context.manifest)
         contents.shouldSyntacticSanityCheck()
         val expectedContents = """
-public class ExampleClient: AWSClientRuntime.AWSServiceClient {
+public final class ExampleClient: AWSClientRuntime.AWSServiceClient, Sendable {
```

Comment:
Rather than explicitly mark every client, we should just add `Sendable` conformance to `ClientRuntime.Client` protocol